### PR TITLE
Rate limit: Error message

### LINKED ISSUE #1024: Add message to web page when limiter is reached
When a user has attempted to log in 10 times in an hour, they are blocked from logging in. Atm though it just looks like the website broke, so we need to display some information to the users

### PR DESCRIPTION
**Pull request by zishanmirza**
Rate limit: Error message

This patch displays an error message when the user makes too many authentication requests.

Fixes #1024.

- [X] Tests passing
- [X] Black formatting
- [ - ] Migrations for any changes to the database schema
- [X] Rebase/merge the `dev` branch
- [x] Note in the CHANGELOG

### CODE REVIEW

File: dds_web/templates/error.html [hunk 1]
```diff
@@ -8,8 +8,12 @@
 {% block body %}
 
 <p>
+{% if error_code == 429 %}
+    API returned an error: Too many authentication requests in one hour
+{% else %}
```
Comment (inaod568) [inline dds_web/templates/error.html:13]:
```suggestion
{% if error_code == 429 %}
    Too many authentication requests in one hour. Try again in a while.
{% else %}
```

**Review by inaod568 — CHANGES_REQUESTED** [review]

File: dds_web/templates/error.html (hunk 1 above)
Comment (zishanmirza) [reply dds_web/templates/error.html]: I have updated the error message.

**Comment by inaod568** [comment]
@zishanmirza Can you add a row in the changelog?

**Comment by zishanmirza** [comment]
I have updated CHANGELOG.md.

**Review by inaod568 — APPROVED** [review]

**Pull request merged**